.codecov.yml

```diff
@@ -475,7 +475,7 @@ coverage:
         flags:
         - sap_hana
       SNMP:
-        target: 50
+        target: 30
```

Comment:
Could you add some description to explain why we are lowering the coverage target?

Feel free to use similar wording as https://github.com/DataDog/integrations-core/pull/15129